### ISSUE TITLE
To Enable Right Click for the Editor

### ISSUE THREAD

**Issue by itssgaurav** [suyati/line-control#42]
Right click is not enabled for the Editor. How can we enable it ? Tried using the "oncontextmenu" not working ?

**Comment by xOneca** [comment]
What element are you attaching the event listener to? Remember that the `textarea` is hidden and replaced by a `div`.

**Comment by itssgaurav** [comment]
I have kept an Id for the "Editor-editor" class in the Editor.js File and for that id setting the 'contextmenu : "true" ' in the jsp. Also tried editing in the .js file for "Editor-editor" class. Doesn't work.

**Comment by xOneca** [comment]
You don't need to change source code for the editor. I have tried the following lines to load the editor in the HTML page and it worked for me:

```js
var $editor = $("#txtArea").Editor();
$editor.on("contextmenu", function(){ alert("Context!"); });
```

**Comment by AlleyLuse** [comment]
I am using LineControl 1.1.0 Last Updated 6/11/2015. I have the same problem, i want to Enable Right Click for the Editor. I try to add code which ( xOneca commented on 24 Aug ) but it is fail. I am using chrome to do testing. I want to enable right click on the div editor area, do you have any solution?

**Comment by xOneca** [comment]
You are right, @AlleyLuse. `$.Editor()` doesn't return editor context, so `$editor` is useless. Can't remember how did I test that... :innocent:

You can use this instead to attach the event (digging in the implementation):

```js
$("#txtArea").data("editor").on("contextmenu", function(){ alert("Context!"); });
```

I can't test it on Chrome right now, but I can confirm that it works on LineControl 1.1.0 with Firefox 42.

**Comment by prharish07** [comment]
can any one say me how to provide a muliple editors on a single page with working  individual image upload icons
